README.md

```diff
@@ -79,6 +84,13 @@ m6_web_log_bridge:
         channel: my_channel_to_log # monolog channel, optional, default 'log_bridge'
 ```
 
+Routes support multiples formats :
+```yaml
+routes: ['my_route'] # Add only this route
+routes: ['my_route', '!excluded_one'] # Add the first route and exclude the second
```

Comment:
This doesn't seem to be the below behavior.
You take into account excluded routes only if you have no route to add.

Reply:
You can only use `routes` or `route` not both at the same time